src/databricks/labs/ucx/install.py

```diff
@@ -371,10 +372,24 @@ def _configure_new_installation(self, default_config: WorkspaceConfig | None = N
             webbrowser.open(ws_file_url)
         return config
 
-    def _config_table_migration(self, spark_conf_dict) -> tuple[int, int, dict]:
+    def _config_table_migration(self, spark_conf_dict) -> tuple[int, int, dict, bool]:
         # parallelism will not be needed if backlog is fixed in https://databricks.atlassian.net/browse/ES-975874
+        managed_to_external_choices = {
+            "[Create managed hive_metastore table as external UC table. This option will "
```

Comment:
can we adjust the wording to `Convert MANAGED tables to EXTERNAL tables and ...`